src/client/versionChecking.js

```diff
@@ -10,7 +10,6 @@ module.exports = function (client, options) {
     try {
       parsed = JSON.parse(message.reason)
     } catch (error) {
-      client.emit('error', error)
```

Comment:
Why do we want to ignore this error?

Reply:
Related to https://github.com/PrismarineJS/node-minecraft-protocol/issues/1290 

The code below checks if the kick message contains an outdated version message, but it's essential. And since the `reason` field is actually a JSON chat message it's still valid for the server to send a plain string (ie "hello world" as opposed to {"text":"hello world"}) as opposed to JSON, as the chat parsing code will accept it. So to correctly parse it we would have to import prismarine-chat and parse it with that.